From 14e34eeeaa8bcae8e7f704fc88526cfa24567890 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Mon, 31 Dec 2018 10:56:12 +0200
Subject: [PATCH] [SW-2751] preboot: print the SPL image offset

Print the SPL image offset that is going to be loaded.

Change-Id: I693764f510217fef80c35c8a3df66d1b5fc40226
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/preboot.c b/preboot.c
index c481acb..f6b2e61 100644
--- a/preboot.c
+++ b/preboot.c
@@ -207,7 +207,9 @@
 
 	preboot_puts_raw("Preparing to run SPL ");
 	preboot_puts_raw(spl_image);
-	preboot_puts("\n");
+	preboot_puts_raw(" (offset: 0x");
+	preboot_puth_raw(image_base_offset, false);
+	preboot_puts(")\n");
 
 	*spl_off = image_base_offset + spl_offset;
 	writel((image_base_offset + uboot_offset), mmPSOC_GLOBAL_CONF_SCRATCHPAD_30);
